app/assets/stylesheets/components/tags.scss

```diff
@@ -1,14 +1,8 @@
 @import '../config/import';
 
 .crayons-tag {
-  --tag-bg: var(--base-a5);
-  --tag-prefix: var(--base-a60);
-
-  --tag-bg-hover: var(--base-a5);
-  --tag-prefix-hover: var(--base);
```

Comment:
These should have been global anyway so I moved them to more appropriate places.